### DISCUSSION TITLE
Incorrect Avatar Documentation

### DISCUSSION THREAD

**Discussion by dazziola** [twilio-labs/paste#749] [General] [answered]
The docs refer to Avatar as being available from the core library. It appears as if this is not the case at this point in time. There is no exported member called Avatar.

![Screen Shot 2020-09-10 at 3 33 48 PM](https://user-images.githubusercontent.com/1335585/92747314-faa70c00-f37b-11ea-9a92-e3b94798b9d0.png)

In order to use it, I had to `yarn add @twilio-paste/avatar`.

Are the docs just incorrect, or should it be available in core?

**Comment by dazziola** [comment]
Further to the doc issues above with Avatar, it appears as though it's default Background Color of type token `colorBackgroundUser` is not being set and is throwing a console warning:

![Screen Shot 2020-09-10 at 3 48 12 PM](https://user-images.githubusercontent.com/1335585/92748590-37273780-f37d-11ea-9fc9-92ba5840347d.png)

As it stands, the Avatar component is quite unusable (especially on darker backgrounds) as it doesn't have a base BG color, and the dark text doesn't work with darker backgrounds.

![Screen Shot 2020-09-10 at 3 52 34 PM](https://user-images.githubusercontent.com/1335585/92749527-18757080-f37e-11ea-9385-3c2964982406.png)

**Comment by richbachman** [comment, marked as answer]
HI @dazziola. Thanks for posting this discussion thread. This is most likely an issue with out of date dependencies. Will you verify you have the most up to date Paste packages installed? You can see a full list here: https://paste.twilio.design/getting-started/engineering#install-paste-system-packages
